src/main/java/peppa/Parser.java

```diff
@@ -0,0 +1,231 @@
+package peppa;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+import peppa.commands.Command;
+import peppa.commands.DeadlineCommand;
+import peppa.commands.DeleteCommand;
+import peppa.commands.EventCommand;
+import peppa.commands.ExitCommand;
+import peppa.commands.FindCommand;
+import peppa.commands.IncorrectCommand;
+import peppa.commands.InvalidCommand;

```

Comment:
Would it be better to make IncorrectCommand and InvalidCommand exceptions too under PeppaExceptions and not as commands?  

There wouldn't be much changes to existing IncorrectCommand and InvalidCommand java files since it prints its error message only iirc.